symfony/plugins/orangehrmCorePlugin/test/fixtures/testcases/AboutOrganizationTestCase.yaml

```diff
@@ -13,7 +13,7 @@ GetOne:
     query: [ ]
     data:
       companyName: 'Orange HRM'
-      version: '5.0'
+      version: 'OrangeHRM OS 5.0-beta'
       numberOfActiveEmployee: 3
       numberOfPastEmployee: 0
```

Comment:
want output like this
```yaml
      companyName: 'OrangeHRM'
      productName: 'OrangeHRM OS'
      version: '5.0-beta'
      numberOfActiveEmployee: 3
      numberOfPastEmployee: 0
```
we may have to update this test (only version attribute) in each release?

Reply:
done